### ISSUE TITLE
Find out from NumFocus how much funds our Binder donate account has

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/team-compass#646]
A year ago we set up a "donate to Binder" link through NumFocus at this page:

https://numfocus.salsalabs.org/donate-to-binder

We then link to that page in our 'support binder' page here:

https://mybinder.readthedocs.io/en/latest/about/support.html#support-the-project

However we don't know how to check this account for any funds, or how to spend them.

### Tasks

- [x] Ask NumFocus how to access the account page for these funds, and how to spend them.
- [x] Get a response from them
- [ ] Document it in the compass and report back here

**Comment by choldgraf** [comment]
### We have about $1,500 in this account

Just heard back from Lisa, there's about $1,500 in it over the last year. Not too bad, considering that we never advertised it! I've asked for clarification on how we can actually _use_ this funding.

**Comment by choldgraf** [comment]
I have a PR to document this account and how to file a reimbursement here:

- https://github.com/jupyterhub/team-compass/pull/651

tl;dr: If we want to use these funds we should pay for things ourselves, and then file a reimbursement later with NumFocus via the instructions here.
